### DISCUSSION TITLE
Add new currency: Ethereum

### DISCUSSION THREAD

**Discussion by kramrm** [grafana/grafana#37662] [General]
Request: Built in unit support for Ethereum.  There are other crypto units included currently, and Ethereum is one of the more popular networks.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
